layouts/partials/episode-module.html

```diff
@@ -1,116 +1,89 @@
-    </ul>
+    <div class="episode-image">
+        <img src="/images/{{ .context.Scratch.Get "imagePath" }}" srcset="/images/{{ .context.Scratch.Get "imagePathHighRes" }} 2x" alt="">
+    </div>
 
     <div class="episode-player">
         <div class="episode-player-wrapper">
-            <iframe title="Listen to {{ .Title }}." class="lazy-load" frameborder="0" height="200px" scrolling="no" seamless src="" data-src="https://embed.simplecast.com/{{ .Params.embed }}?color=f5f5f5" width="100%"></iframe>
+            <iframe title="Listen to {{ .context.Title }}." class="lazy-load" frameborder="0" height="200px" scrolling="no" seamless src="" data-src="https://embed.simplecast.com/{{ .context.Params.embed }}?color=f5f5f5" width="100%"></iframe>
             <noscript>
-                <iframe title="Listen to {{ .Title }}." frameborder="0" height="200px" scrolling="no" seamless src="https://embed.simplecast.com/{{ .Params.embed }}?color=f5f5f5" width="100%"></iframe>
+                <iframe title="Listen to {{ .context.Title }}." frameborder="0" height="200px" scrolling="no" seamless src="https://embed.simplecast.com/{{ .context.Params.embed }}?color=f5f5f5" width="100%"></iframe>
             </noscript>
         </div>
     </div>
 
-    <section class="subscribe-links-wrapper episode-view-single">
-        <h4>Subscribe:</h4>
-
-        {{ partial "subscribe-links.html" (dict "itunesTracking" (.Scratch.Get "itunesTracking") "rpTracking" (.Scratch.Get "rpTracking") "title" .Title) }}
-    </section>
+    <ul class="episode-actions">
+        <li><button type="button" class="episode-subscribe mzp-c-button mzp-t-small mzp-t-product" data-subscribe-links="episode-{{ .context.Params.number }}-subscribe-links">Subscribe</button></li>
+        <li><a class="episode-action icon icon-twitter" data-service="twitter" href="https://www.twitter.com/intent/tweet?url={{ .context.Permalink }}&amp;text=IRL Podcast: {{ .context.Title }} {{ .context.Scratch.Get "host" }}" data-link-type="social share" data-link-name="Twitter">Share on Twitter</a></li>
+        <li><a class="episode-action icon icon-facebook" data-service="facebook" href="https://www.facebook.com/sharer/sharer.php?u={{ .context.Permalink }}" data-link-type="social share" data-link-name="Facebook">Share on Facebook</a></li>
+        <li class="js-action-copy">
+            <a class="episode-action icon icon-copylink" data-service="copylink" href="{{ .context.Permalink }}" data-link-type="social share" data-link-name="copylink" data-clipboard-text="{{ .context.Permalink }}">Copy Link</a>
+            <div class="episode-copy-ok">URL copied to clipboard</div>
+        </li>
+        <li><a class="episode-action icon icon-download" data-service="download" href="{{ .context.Params.mp3 }}" rel="external noopener noreferrer">Download MP3</a></li>
+    </ul>
 
     <div class="episode-description">
         <p>
```

Comment:
It looks like we don't need this wrapper `<p>` tag, as `markdownify` seems to generate them automatically. Results in nested `<p>` tags, which doesn't actually hurt anything but I believe is invalid markup.

Reply:
✅